packages/flutter_tools/lib/src/build_system/targets/android.dart

```diff
@@ -307,15 +311,29 @@ class AndroidAotBundle extends Target {
 
   @override
   Future<void> build(Environment environment) async {
-    final File outputFile = environment.buildDir
-      .childDirectory(_androidAbiName)
-      .childFile('app.so');
+    print(name);
+    final Directory buildDir = environment.buildDir.childDirectory(_androidAbiName);
+    // final Directory moduleDir = environment.projectDir.cchildDirectory(_androidAbiName);
     final Directory outputDirectory = environment.outputDir
       .childDirectory(_androidAbiName);
     if (!outputDirectory.existsSync()) {
       outputDirectory.createSync(recursive: true);
     }
-    outputFile.copySync(outputDirectory.childFile('app.so').path);
+    for (FileSystemEntity file in buildDir.listSync()) {
```

Comment:
whats going on here?

Reply:
Copying all produced .so files + manifest instead of just app.so